src/BlockEntities/BlockEntity.h

```diff
@@ -103,7 +103,7 @@ class cBlockEntity
 	// tolua_end
 	
 	/** Called when a player uses this entity; should open the UI window */
-	virtual void UsedBy( cPlayer * a_Player) = 0;
+	virtual bool UsedBy( cPlayer * a_Player) = 0;
```

Comment:
Again, document the return value. Need to do that for all non-overriding functions.